examples/assets/README.md

```diff
@@ -3,3 +3,5 @@
 This directory contains various files for the example notebooks.
 
 - `mp-978534.poscar` contains a SiGe structure from [https://materialsproject.org/materials/mp-978534/](https://materialsproject.org/materials/mp-978534/).
+- `bash_workflow-template.json` contains the template JSON schema of Mat3ra workflow.
```

Comment:
I overlooked one suggestion during my review. The added file is `examples/assets/bash_workflow_template.json`, so it needs an underscore instead of a dash:
```suggestion
- `bash_workflow_template.json` contains the template JSON schema of Mat3ra workflow.
```